src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -14,6 +14,21 @@ class PDOIntegration extends Integration
 
     const CONNECTION_TAGS_KEY = 'connection_tags';
 
+    const DB_DRIVER_TO_SYSTEM = [
```

Comment:
The PHP files listed in `_files_integrations.PHP5.php` must all still confirm to PHP 5.4 syntax, because they're still distributed from the master branch. We need to eventually fully separate these, but for now, arrays are not allowed in constants, nor is `??`.

Reply:
was not aware of this, thanks for letting me know. This is should be all set.